acceptance/config/git/options.rb

```diff
@@ -1,7 +1,5 @@
 {
   :install => [
-    'facter#2.x',
-    'hiera#1.3.4',
     'puppet',
```

Comment:
maybe expand out `puppet` to `https://github.com/puppetlabs/puppet` so it's less magical